src/ucp/core/ucp_worker.c

```diff
@@ -3071,6 +3070,11 @@ static UCS_F_NOINLINE unsigned ucp_worker_do_keepalive_progress(void *arg)
     ucs_time_t now;
     ucp_ep_h ep;
 
+    /* Must be called once per 'UCP_WORKER_KEEPALIVE_ITER_SKIP' iterations */
+    ucs_assertv(((worker->keepalive.iter_count - 1) %
```

Comment:
seems redundant, it's not like this function really requires this condition in order to work properly

Reply:
removed